pkg/cache/clusterqueue_test.go

```diff
@@ -98,206 +98,281 @@ func TestFitInCohort(t *testing.T) {
 	cases := map[string]struct {
 		request            resources.FlavorResourceQuantitiesFlat
 		wantFit            bool
-		cq                 *ClusterQueueSnapshot
+		flavorNames        []string
```

Comment:
nit: it looks like all the tests use "f1", and (most of the time) "f2"

Please delete this field, and just harcode those flavors in the test fixture